docs/_docs/07-dg-code-structure.md

```diff
@@ -6,3 +6,28 @@ last_modified_at: 2020-06-20
 toc: true
 ---
 
+Hyperspace code is written in Scala and it is using 'sbt' as the build tool. Below is the list of folders in the code base hierarchy:
+### docs
+“docs” folder should be the starting point for looking into Hyperspace code base. It contains useful documentations on different aspects of Hyperspace project, including coding and formatting guidelines, Hyperspace configurations and information on how to build the project in various environments. 
+### src
```

Comment:
Can we move `src` to the top?
Also can you add some new lines around ### to make it more readable?

Reply:
Done.